user profile page for admins

Louie112 opened the pull request.

Louie112 committed:
user profile page for admins

GeeCracker reviewed (changes requested):
- [ ] this needs to be a variable url like the quiz page where the var is the specified user id
- [ ] formatting needs another pass - lmk if you want help, we can work on it on friday
- the code itself is clean asf tho

Louie112 committed:
updated

-changed to variable url
-formatting improved to accommodate different screen sizes.

GeeCracker committed:
little touch-ups
